contexts/citizenship-v1.jsonld

```diff
@@ -0,0 +1,23 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "id": "@id",
+    "type": "@type",
+    "ctzn": "https://w3id.org/citizenship#",
+    "schema": "http://schema.org/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+    "Person": "ctzn:Person",
```

Comment:
```suggestion
    "Person": "schema:Person",
```

Reply:
We are missing the following class.:

  "PermanentResidentCard": "ctzn:PermanentResidentCard"

Reply:
@msporny That is not defined in the vocab. I can go ahead and add it to the context though.

Reply:
Can someone provide clarification on this point?

Reply:
schema:person is correct per discussion.  Resolved.